cms/templates/group_configurations.html

```diff
@@ -47,30 +49,40 @@ <h1 class="page-header">
 
   <div class="wrapper-content wrapper">
     <section class="content">
+
       <article class="content-primary" role="main">
+        % if should_show_enrollment_track:
```

Comment:
I'd love to see this template just iterate over user partitions (excluding A/B experiments, which are special cased) instead of having if blocks for enrollment tracks and content groups.